go/pkg/bertymessenger/handlers.go

```diff
@@ -295,10 +295,11 @@ func (h *eventHandler) accountContactRequestOutgoingEnqueued(gme *protocoltypes.
 	contactPKBytes := ev.GetContact().GetPK()
 	contactPK := b64EncodeBytes(contactPKBytes)
 
+	// FIXME: Discuss about removing this whole Metadata unmarshaling by setting `ShareableContact.metadata` type directly to `ContactMetadata`.
 	var cm messengertypes.ContactMetadata
-	err := proto.Unmarshal(ev.GetContact().GetMetadata(), &cm)
-	if err != nil {
-		return err
+	if err := proto.Unmarshal(ev.GetContact().GetMetadata(), &cm); err != nil {
```

Comment:
did this happen while you were testing?

it's maybe not "critical", it's really strange if it happens and I'm not sure it's a good thing to ignore it at this stage -> need additional reviews

Reply:
Yes this does happend, this is mini using a pure string instead of a protobuf struct.